other/edge-server/client/react-client/README.md

```diff
@@ -0,0 +1,13 @@
+## Edge Server React Client
+
+TODO: add info about how this client fits into the Edge Server template
```

Comment:
Same Q as above. We should probably either populate details here or remove this README.md from the branch.